packages/common/src/account/kms/kmsKeyToAddress.test.ts

```diff
@@ -102,4 +107,35 @@ describe("createKmsAccount", () => {
 
     expect(valid).toBeTruthy();
   });
+
+  it("can execute transactions", async () => {
+    const adminClient = createWalletClient({
+      chain: foundry,
+      transport: http(anvilRpcUrl),
+      account: privateKeyToAccount("0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80"),
+    });
+
+    const kmsClient = createWalletClient({
+      chain: foundry,
+      transport: http(anvilRpcUrl),
+      account,
+    });
+
+    // Fund the KMS account from the admin
+    let balance = await getBalance(adminClient, { address: account.address });
+    expect(balance).toEqual(0n);
+
+    let tx = await adminClient.sendTransaction({ to: account.address, value: parseEther("1") });
+    await waitForTransaction(tx);
+
+    balance = await getBalance(adminClient, { address: account.address });
+    expect(balance).toEqual(1000000000000000000n);
```

Comment:
we're using a mix of tree shakable and not-tree shakable actions, would be nice to consolidate (ideally on tree shakable)

and if we're doing all tree shakable, might make sense to just use `createClient` not `createWalletClient`